cityofaustin/atd-vz-data
# 10230 multiple partners

## Associated issues

This PR allows users to select multiple coordination partners for a recommendation instead of just one. It required restructuring the back-end database using a many to many relationship between partners and recommendations.

Following this PR, coordination partner information will need to be migrated from the `recommendations` table to the new `recommendations_partners` table in issue [#10462](https://github.com/cityofaustin/atd-data-tech/issues/10462)

Closes cityofaustin/atd-data-tech/issues/10230

## Testing
**URL to test:** 

This can be tested locally or with the Netlify preview: https://deploy-preview-1110--atd-vze-staging.netlify.app/#/crashes

**Steps to test:**
1. Click on Crashes tab.
2. Filter crashes by ATD death count (Recommendations section only shows up on crashes that had a fatality). Click on any crash with a fatality.
4. Scroll to Fatality Review Board Recommendations section. Add multiple partners, remove partners, refresh, make sure your changes stick.
5. You can also play around with the rest of the section to make sure everything works together.

---
#### Ship list
